drivers/classic/src/get_global_param.c

```diff
@@ -1029,16 +1169,18 @@ get_global_param(FILE *gp)
             if (param_set.FORCE_FORMAT[i] == MISSING) {
                 log_err("FORCE_FORMAT%d: %d. Need to specify the FORCE_FORMAT "
                         "(ASCII or BINARY) for forcing file %d.", i,
-                        param_set.FORCE_FORMAT[i], i);
+                        param_set.FORCE_FORMAT[i],
+                        i);
```

Comment:
There are a number of these whitespace only changes that clutter the commit. I don;t understand why uncrustify would have introduced these. Since the original code was run through uncrustify already, we should not include these whitespace only changes in the commit. There are others like it. Please remove them (using sourcetree it's easy to revert selected lines only).